2dcontext/tools/gentestutils.py

```diff
@@ -233,7 +233,7 @@ def expand_test_code(code):
                 code)
 
         code = re.sub(r'@assert throws (\S+Error) (.*);',
-                r'assert_throws(new \1(), function() { \2; });',
+                r'assert_throws_js(\1, function() { \2; });',
```

Comment:
The bit right above this probably needs to become `assert_throws_dom("\1", etc`, right?

Reply:
Yes, I will do that when I get to the commit where you did `assert_throws_dom` rewrites.